frontend/packages/client/src/components/Community/ProposalThresholdEditor/ThresholdForm.js

```diff
@@ -32,6 +34,23 @@ export default function ThresholdForm({
             </p>
           </div>
         </div>
+        <Dropdown
+          label="Contract Type"
+          name="contractType"
+          margin="mt-4"
+          options={[
+            {
+              label: 'NFT',
+              value: 'nft',
+            },
+            {
+              label: 'Fungible Token',
+              value: 'ft',
+            },
```

Comment:
I think here we need to add an empty option otherwise if the user selects the field and then want to leave everything empty and check on onlyAuthors....it's not possible:

<img width="570" alt="Screen Shot 2022-08-22 at 13 24 00" src="https://user-images.githubusercontent.com/7526740/185971319-f4b8b59c-0c6c-404d-a7c1-e4c6d026ff9f.png">

maybe: 

````options={[
            {
              label: 'NFT',
              value: 'nft',
            },
            {
              label: 'Fungible Token',
              value: 'ft',
            },
            {
              label: '' or 'None',
              value: '',
            },

Reply:
Added. I think Default makes sense because it just means use whatever the default is and the user doesn't care about that value.